map/templates/map/default.html

```diff
@@ -0,0 +1,48 @@
+{% extends 'post/base.html' %}
+
+{% block content %}
+
+<style>
```

Comment:
We should put style in style.css file, or create the new map.css and put it there.
We can do it later with refactoring though:)